pkg/controller/factory/base_controller.go

```diff
@@ -84,6 +89,9 @@ func QueueKey(name string) string {
 }
 
 func (c *baseController) runPeriodicalResync(ctx context.Context, interval time.Duration) {
+	if reflect.ValueOf(interval).IsZero() {
```

Comment:
reflect? 😱 Why?

Reply:
it seems like in go-1.13 this is the "right" way to check if something is "zero" (iow. not set).

Reply:
No, it's not. Just compare with `time.Duration(0)`. Duration is just an int64.